docs/source/training/training.rst

```diff
@@ -0,0 +1,787 @@
+         ├── beam[4]
+         │   └── beam[4]/power_launched
+         │       └── data: array([    0.    , 21052.6316, 42105.2632, ...,     0.    ,     0.    ,     0.    ])
+         ├── beam[5]
+         │   └── beam[5]/power_launched
+         │       └── data: array([    0.    , 21052.6316, 42105.2632, ...,     0.    ,     0.    ,     0.    ])
+         ├── beam[6]
+         │   └── beam[6]/power_launched
+         │       └── data: array([    0.    , 21052.6316, 42105.2632, ...,     0.    ,     0.    ,     0.    ])
+         ├── beam[7]
+         │   └── beam[7]/power_launched
+         │       └── data: array([    0.    , 21052.6316, 42105.2632, ...,     0.    ,     0.    ,     0.    ])
+         ├── beam[8]
+         │   └── beam[8]/power_launched
+         │       └── data: array([    0.    , 21052.6316, 42105.2632, ...,     0.    ,     0.    ,     0.    ])
+         ├── beam[9]
+         │   └── beam[9]/power_launched
+         │       └── data: array([    0.    , 21052.6316, 42105.2632, ...,     0.    ,     0.    ,     0.    ])
+         └── time: array([  0.    ,  42.1053,  84.2105, ..., 715.7895, 757.8947, 800.    ])
+
+
+Exercise 5b: Exporting different Data Dictionary versions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. md-tab-set::
+   .. md-tab-item:: Exercise
+
+      Repeat the previous exercise, but this time, before exporting change the data dictionary 
+      version to **3.42.0** in the `Edit Global Properties` tab, and save the configuration.
+      Ensure you enter a different from previous exercise. Again, print the IDS in your terminal, what has changed?
+
+   .. md-tab-item:: Solution
+
+      You should see that the data dictionary version of the IDS has changed to '3.42.0':
+
+      .. code-block:: bash
+
+         ec_launchers
+         ├── ids_properties
+         │   ├── homogeneous_time: 1
+         │   └── ids_properties/version_put
+         │       ├── data_dictionary: '3.42.0'
+         │       ├── access_layer: '5.4.3'
+         │       └── access_layer_language: 'imas 2.0.1'
+         ...
+
+Exercise 5c: Exporting from the CLI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. md-tab-set::
+   .. md-tab-item:: Exercise
+
+      You can also export a configuration using the CLI. Try exporting your configuration
+      using the same settings with the CLI command. Print the IDS afterwards, is it the 
+      same as before?
+
+      .. hint::
+
+         Each CLI command has a help page which can be printed by supplying the ``--help``
+         flag, for example:
+
+         .. code-block:: bash
+
+            waveform-editor --help 
+
+         Detailed instructions on how to use the CLI can be found :ref:`here <cli>`.
+
+   .. md-tab-item:: Solution
+
+      Export the configuration using:
+
+      .. code-block:: bash
+
+         waveform-editor export-ids <example YAML> <your URI> --linspace 0,800,20
```

Comment:
Funny highlighting: everything is grey except the first 0 in 0,800,20 (which is red)

<img width="1134" height="153" alt="Image" src="https://github.com/user-attachments/assets/bbe56561-d475-4aab-85a6-91416b6119a2" />